intercept/requests.go

```diff
@@ -52,12 +52,12 @@ func handleRequest(preq *pendingRequest) {
 		providedResp, err = http.ReadResponse(providedResponseBuffer, preq.originalRequest)
 		if err != nil {
 			//TODO check this error and hijack connection to send raw bytes
-			log.Println("Error during provided response parsingh")
+			lg.Errorf("Error during provided response parsingh\n")
 		}
 		status.addRawEditedResponse(preq.id, editedRequestDump)
 	default:
 		//TODO implement this
-		log.Println("Not implemented yet")
+		lg.Errorf("Not implemented yet\n")
```

Comment:
You could use lg.Error() instead of Errorf with a \n at the end.

Reply:
Yes, I'll change it, just missed it